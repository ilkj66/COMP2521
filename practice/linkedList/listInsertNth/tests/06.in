42
16
0
