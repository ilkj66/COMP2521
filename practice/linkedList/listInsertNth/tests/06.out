Enter list: Enter value to insert: Enter position to insert at: 
Original list: [42] -> X
List after inserting 16 at position 0: [16] -> [42] -> X
